scripts/backup

```diff
@@ -26,8 +27,8 @@ ynh_print_info --message="Loading installation settings..."
 
 app=$YNH_APP_INSTANCE_NAME
 
-final_path=$(ynh_app_setting_get --app="$app" --key=final_path)
-domain=$(ynh_app_setting_get --app="$app" --key=domain)
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
+domain=$(ynh_app_setting_get --app=$app --key=domain)
```

Comment:
Please keep double quotes and such

Reply:
I did a PR of example_ynh to add those, so I accept your changes to comply this app to example_ynh.

Reply:
Just applying example_ynh